WordPress/Classes/Services/PostService.m

```diff
@@ -123,33 +123,66 @@ - (void)syncPostsOfType:(PostServiceType)postType
                 forBlog:(Blog *)blog
                 success:(PostServiceSyncSuccess)success
                 failure:(PostServiceSyncFailure)failure
+{
+    [self syncPostsOfType:postType
+              withOptions:options
+                  forBlog:blog
+              loadedPosts:@[].mutableCopy
```

Comment:
This is super nitpicky.  Our [objc style guide](https://github.com/wordpress-mobile/objective-c-style-guide) calls for dot notation only for `@proprties`.  Method calls in particular we like to retain the bracket syntax: `[@[] mutableCopy]`.